attachment_zipped_download/models/ir_attachment.py

```diff
@@ -28,9 +28,18 @@ def _create_temp_zip(self):
         zip_buffer = BytesIO()
         with zipfile.ZipFile(zip_buffer, "a", zipfile.ZIP_DEFLATED, False) as zip_file:
             for attachment in self:
-                zip_file.write(
-                    attachment._full_path(attachment.store_fname), attachment.name
+                # Use attachment read for two purpose (initial implementation wasn't):
+                # * to ensure access write
+                # * do not assume file is stored on filesystem
+                zip_file.writestr(
```

Comment:
Are you sure `writestr` is compatible with all the contents?

Reply:
[Documentation tells that it can receive str or bytes](https://docs.python.org/3/library/zipfile.html#zipfile.ZipFile.writestr), I have compressed various types likes  `.jpg`